### ISSUE TITLE
Duplication of the Dialogi of Seneca

### ISSUE THREAD

**Issue by AlisonBabeu** [PerseusDL/catalog_data#15]
There is an issue with the way the individual Dialogi of Seneca are displaying in the catalog. The system has both "correctly" aggregated these works by their individual STOA numbers, for example see the work record for the De Brevitate Vitate (http://catalog.perseus.org/catalog/urn:cts:latinLit:stoa0255.stoa004- data at https://github.com/PerseusDL/catalog_data/tree/master/mods/latinLit/stoa0255/stoa004) but it has also created a top level work record for the Dialogi under phi 1017.12 (catalog view-http://catalog.perseus.org/catalog/urn:cts:latinLit:phi1017.phi012, files at (https://github.com/PerseusDL/catalog_data/tree/master/mods/latinLit/phi1017/phi012). This has led to the creation of duplicate edition entries for the individual Dialogi under the PHI identifier. Additionally this means that the full list of Dialogi do not appear as separate works under the authority record for Seneca http://catalog.perseus.org/catalog/urn:cts:latinLit:phi1017.

**Comment by AlisonBabeu** [comment]
These issues appear to have been created by some issues with the authority records for Seneca, which I have now hopefully fixed in the CiteTables. I'm leaving this issue open until the next ingest to see if the fix works.

**Comment by annakrohn** [comment]
So just to confirm, we've removed the MADS file for the Pseudo Seneca that was at PrimaryAuthors/S/Seneca/Pseudo-Seneca/Octavia/n83-230210.xml.mads.xml entirely?

**Comment by AlisonBabeu** [comment]
Yes. I created a new top level authority record for Pseudo-Seneca that is currently in catalog_pending.
https://github.com/PerseusDL/catalog_pending/tree/master/mads/Pseudo-Seneca

**Comment by annakrohn** [comment]
Okay, so there is only one Pseudo-Seneca now.  Where as before there were two, the PrimaryAuthors/S/Seneca/Pseudo-Seneca/Octavia/n83-230210.xml.mads.xml and the PrimaryAuthors/S/Seneca/Pseudo-Seneca/nr99-11706.xml.mads.xml.

**Comment by AlisonBabeu** [comment]
Yes. I tried to eliminate all others in my purge.
